### PR TITLE
add user registration via file2chan

### PR DESCRIPTION
**Pull request by dievri**

### CODE REVIEW

**Comment by dievri** [comment]
@dievri please merge

**Comment by dievri** [comment]
@dievri @metacoma verify

File: profile [hunk 1]
```diff
@@ -21,6 +21,7 @@ cmd_file := $dir^/tmp/cmd
 
 test -d $dir/export || mkdir -p $dir/export
 test -d $dir/tmp || mkdir -p $dir/tmp
+test -d /users || mkdir -p /users
```
Comment (metacoma) [inline profile]: Pass /users from host, through docker volume

**Review by metacoma — COMMENTED** [review]
https://github.com/9mine/9mine-auth/pull/1/commits/7cc763afb0dab8237d1bbe1d23975cc888f68793#diff-7d97481b1fe66f4b51db90da7e794d9fR24

Pass /users from the linux host, through docker volume

**Comment by metacoma** [comment]
@dievri Pass /users from the linux host, through docker volume

https://github.com/9mine/9mine-auth/commit/7cc763afb0dab8237d1bbe1d23975cc888f68793#diff-7d97481b1fe66f4b51db90da7e794d9fR24

**Review by metacoma — APPROVED** [review]

**Comment by metacoma** [comment]
@metacoma LGTM, merge

**Comment by metacoma** [comment]
@dievri LGTM

**Pull request merged**